src/App.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <v-app>
+    <v-main class="main">
+      <div class="fade_container">
+        <router-view v-slot="{ Component }">
+          <transition name="fade">
+            <component :is="Component" />
+          </transition>
+        </router-view>
+      </div>
+      <Footer />
+    </v-main>
+  </v-app>
+</template>
+
+<script lang="ts">
```

Comment:
@yannde 
Is `setup` script still unstable? 
I thought `setup` script would be good because the code would be concise.

https://vuejs.org/api/sfc-script-setup.html

Reply:
@jinmayamashita 
Yes, we can test using the setup script instead. I suggest making a setup branch once the base is merged into main.

Reply:
@jinmayamashita 
I've converted to script setup
The only thing that's not done yet is [accessing i18n](https://github.com/monstar-lab-oss/vuejs-boilerplate/blob/388d99b82d1c459993182cb1c18e27f04664bf4d/src/views/Main/Main.vue#L28) inside the script